src/components/topic/TopicService.js

```diff
@@ -71,10 +71,15 @@ goog.require('ga_permalink');
         });
 
         // Returns a promise that is resolved when topics are loaded
-        this.getTopics = function() {
+        this.loadConfig = function() {
           return topicsP;
```

Comment:
We probably have a race condition here. I try to explain.

When the loadConfig promise is resolved, 2 things happen:
A) init the gaTopic service (set topics and topic variables on line 66), which leads to the first `gaTopicChange` event being triggered
B) init other components. In those 'then' functions, we attach to `gaTopicChange` event.

If the 'then' of A is called before the 'then' of B, the component in B will miss the first `gaTopicChange` event, no?